_submissions/round-12/1/2015-02-26-wrapping-fortran-func-subs-to-python.md

```diff
@@ -0,0 +1,16 @@
+---
+date: 2015-02-26
+round: Round 12
+title: Wrapping Fortran functions/subroutines to Python
```

Comment:
Ok, I thought it could be useful for the function creations in python. At
some point the execution time of numerical functions is important too, but
if it not the case I will resubmit it tonight.
Bests, Horacio.

On Fri, Feb 27, 2015 at 4:32 AM, Greg Wilson <notifications@github.com>
wrote:

> In
> _submissions/round-12/1/2015-02-26-wrapping-fortran-func-subs-to-python.md
> <https://github.com/swcarpentry/training-course/pull/254#discussion_r25486166>
> :
>
> > @@ -0,0 +1,16 @@
> > +---
> > +date: 2015-02-26
> > +round: Round 12
> > +title: Wrapping Fortran functions/subroutines to Python
>
> The homework assignment asked for a concept map of part of one of our
> existing lessons, in part to encourage people to look through the material
> we have and start to familiarize themselves with it. Can you please do that
> and resubmit?
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/swcarpentry/training-course/pull/254/files#r25486166>.
>



-- 
Horacio A. Vargas Guzmán, PhD